static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -59,7 +59,9 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
       return;
     }
 
-    await this.getAuthenticators();
+    const authenticators = await this.getAuthenticators();
+    this.setState({authenticators: authenticators});
+
     // Set the error state if there are no authenticators and U2F is on
     if (!this.state.authenticators.length && !disableU2FForSUForm) {
```

Comment:
`componentDidMount()` will only run once on mount, so this part needs to be updated as well:

```suggestion
    if (!authenticators && !disableU2FForSUForm) {
```

Reply:
wait do I need to keep the `.length` part?

Reply:
@schew2381 ah yea that too.

Reply:
alright done